code/controllers/subsystem/security_level.dm

```diff
@@ -56,6 +56,13 @@ SUBSYSTEM_DEF(security_level)
 		SSshuttle.emergency.modTimer(old_shuttle_call_time_mod)
 		SSshuttle.emergency.modTimer(selected_level.shuttle_call_time_mod)
 
+//SKYRAT EDIT ADDITION START- In the event of an orange alert, change the airlock permissions
+	if(current_security_level.name == "orange")
```

Comment:
is there a define for the alert level to be used here?

Reply:
```
/datum/controller/subsystem/security_level/proc/set_level(new_level)
	new_level = istext(new_level)
SSsecurity_level.current_security_level = selected_level
```